packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -215,7 +221,7 @@ export class Slider extends FormAssociated<HTMLInputElement>
     private setupTrackConstraints = (): void => {
         this.trackWidth = this.track.clientWidth;
         this.trackMinWidth = this.track.clientLeft;
-        this.trackHeight = this.track.clientHeight;
+        this.trackHeight = this.track.getBoundingClientRect().bottom;
```

Comment:
Let's refactor this to one call to `getBoundingClientRect()` so we can use that to get both bottom and top.

Reply:
done and done